packages/edit-post/src/components/layout/index.native.js

```diff
@@ -150,11 +151,17 @@ class Layout extends Component {
 						style={ toolbarKeyboardAvoidingViewStyle }
 						withAnimatedHeight
 					>
-						{ Platform.OS === 'ios' && <FloatingToolbar /> }
+						{ Platform.OS === 'ios' && (
+							<>
+								<AutocompletionItemsSlot />
+								<FloatingToolbar />
+							</>
+						) }
 						<Header />
 						<BottomSheetSettings />
 					</KeyboardAvoidingView>
 				) }
+				{ Platform.OS === 'android' && <AutocompletionItemsSlot /> }
```

Comment:
Why we have this difference between iOS and Android?

Reply:
If I recall correctly, this is due to differences in the placement of the `FloatingToolbar` component on iOS vs. Android.